### ISSUE TITLE
Destiny tries to create folder named index.jsx

### ISSUE THREAD

**Issue by leodr** [benawad/destiny#73]
With my React project destiny tries to make a folder named `index.jsx` instead of just `index`.

This is my file structure right now:

```
│   data.json
│   index.html
│   index.jsx
│
├───components
│   │   App.jsx
│   │   Contribute.jsx
│   │   CreateProject.jsx
│   │   Donator.jsx
│   │   Donators.jsx
│   │   donatorShape.js
│   │   Expandable.jsx
│   │   FinishedProjects.jsx
│   │   Footer.jsx
│   │   ImageCarousel.jsx
│   │   ImageSection.jsx
│   │   ShareButton.jsx
│   │   ShowMoreButton.jsx
│   │
│   └───ProjectCard
│           Progress.jsx
│           ProjectCard.jsx
│
└───util
        formatMoney.js
```

And this is what destiny suggests: 

```
├──index.jsx
├──index.jsx\shared\App.jsx
├──index\App\CreateProject.jsx
├──index\App\data.json
├──index\App\FinishedProjects.jsx
├──index\App\Footer.jsx
├──index\App\ProjectCard.jsx
├──index\App\ProjectCard\Contribute.jsx
├──index\App\ProjectCard\Donators.jsx
├──index\App\ProjectCard\Donators\Donator.jsx
├──index\App\ProjectCard\ImageSection.jsx
├──index\App\ProjectCard\ImageSection\ImageCarousel.jsx
├──index\App\ProjectCard\Progress.jsx
├──index\App\ProjectCard\ShareButton.jsx
├──index\App\ProjectCard\ShowMoreButton.jsx
├──shared\donatorShape.js
├──shared\Expandable.jsx
└──shared\formatMoney.js
```

**Comment by leodr** [comment]
This is the error message when i try to execute it with `npx destiny -w src/**/*.*`:

```
INFO: Fixing imports.
INFO: Moving files.
(node:16644) UnhandledPromiseRejectionWarning: Error: EEXIST: file already exists, mkdir 'C:\Dev\frontend\crowdfunding-prototype\src\index.jsx'
    at Object.mkdirSync (fs.js:840:3)
    at mkdirsSync (C:\Dev\frontend\crowdfunding-prototype\node_modules\fs-extra\lib\mkdirs\mkdirs-sync.js:31:9)
    at Object.mkdirsSync (C:\Dev\frontend\crowdfunding-prototype\node_modules\fs-extra\lib\mkdirs\mkdirs-sync.js:36:14)
    at moveFiles (C:\Dev\frontend\crowdfunding-prototype\node_modules\destiny\lib\destiny.js:68:19)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async formatFileStructure (C:\Dev\frontend\crowdfunding-prototype\node_modules\destiny\lib\destiny.js:235:5)
    at async run (C:\Dev\frontend\crowdfunding-prototype\node_modules\destiny\lib\destiny.js:754:5)
(node:16644) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process 
on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:16644) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by sQVe** [comment]
Could you please provide an example repository to debug on? It's pretty much the only way for us to know whats going wrong.

**Comment by leodr** [comment]
https://github.com/leodr/destiny-bug-repo

I stripped some files but this is enough to reproduce the issue for me when running `npx destiny -w .`

**Comment by sQVe** [comment]
This is now released with #75. @leodr Please try the new version and get back to us if it works as intended. I'll reopen if it doesn't. 

Cheers for taking the time to create a debug repo :beers: 

Closing.
